src/txkube/test/test_authentication.py

```diff
@@ -143,13 +143,13 @@ def _authorized_request(self, token, headers,
 
         self.patch(
             os, "environ", {
-                b"KUBERNETES_SERVICE_HOST": kubernetes_host,
-                b"KUBERNETES_SERVICE_PORT": b"443",
+                u"KUBERNETES_SERVICE_HOST": kubernetes_host.decode("ascii"),
+                u"KUBERNETES_SERVICE_PORT": u"443",
```

Comment:
This seems to be an unrealistic `patch` for Python 2.  When would you find unicode in `os.environ` on Python 2?